TYPO3.Neos/Resources/Private/Templates/Media/Asset/RelatedNodes.html

```diff
@@ -59,12 +59,12 @@
 							<ul>
 								<f:for each="{documentNode.nodes}" as="relatedNode">
 									<li>
-										<f:if condition="{relatedNode.contextDocumentNode}">
+										<f:if condition="{relatedNode.accessible}">
```

Comment:
changing this break the check if there is even a contextDocumentNode.. it probably needs to check for both actually, because either you don't have access or the node is missing..